tests/test_job.py

```diff
@@ -12,21 +12,17 @@
 import random
 import json
 from contextlib import contextmanager
+from tempfile import TemporaryDirectory
 
 import signac.contrib
 import signac.common.config
-from signac import Project
-from signac.contrib.job import Job
-from signac.common import six
+from signac import Project  # noqa: F401
+from signac.contrib.job import Job  # noqa: F401
```

Comment:
Why is this not used?

Reply:
These imports must be here for tests to pass, despite not being directly used. They are only used indirectly, in the test `eval(repr(x))` -- the name given by `repr` is not fully-qualified (e.g. `signac.Project` or `signac.contrib.job.Job`) so it must imported from that module. In `freud` I made the choice to use fully-qualified names (but similarly, that relies on using `import freud` instead of `from freud.box import Box`).

Reply:
Ok, that's fine, just checking.